SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRepNotify/SpatialTestRepNotify.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SpatialFunctionalTest.h"
+#include "SpatialTestRepNotify.generated.h"
+
+UCLASS()
+class SPATIALGDKFUNCTIONALTESTS_API ASpatialTestRepNotify : public ASpatialFunctionalTest
+{
+	GENERATED_BODY()
+
+public:
+	ASpatialTestRepNotify();
+
+	virtual void BeginPlay() override;
+
+	void GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const override;
+
+	bool bOnRepTestInt1Called;
+
+	bool bOnRepTestInt2Called;
+
+	int32 OldTestInt3;
+
+	int32 OldTestInt4;
+
+	TArray<int32> OldTestArray;
+
+	UPROPERTY(ReplicatedUsing = OnRep_TestInt1)
+	int32 TestInt1;
```

Comment:
nit - as the key difference here is the rep condition it would be good to have that included in the variable name

Reply:
+1